study-builder/studies/rarex/study.conf

```diff
@@ -149,7 +123,7 @@
               user.studies["rarex"].forms["PREQUAL"].questions["SELF_COUNTRY"].answers.hasOption("PR")
               && user.studies["rarex"].forms["PREQUAL"].questions["SELF_CURRENT_AGE"].answers.value() < 21
             ) || (
-              user.studies["rarex"].forms["PREQUAL"].questions["SELF_COUNTRY"].answers.hasAnyOption("GU", "VI", "MP", "AS")
+              !user.studies["rarex"].forms["PREQUAL"].questions["SELF_COUNTRY"].answers.hasAnyOption("US", "CA", "PR")
```

Comment:
This is the key change:
* If we're in US/CA/PR, then we should follow whatever those rules are.
* Only if it's not US/CA/PR, then we do check for 18.

This ensures we handle the case where, e.g. you're in US+Alabama but you're not 19.

Reply:
Are we sure Guam, Virgin Islands and other American territories should be treated differently than the territory of Puerto Rico?

Reply:
[See Slack comment](https://broadinstitute.slack.com/archives/C01A6HWK3PX/p1620679623109700?thread_ts=1620671795.091400&cid=C01A6HWK3PX).

![Screen Shot 2021-05-12 at 3 49 04 PM](https://user-images.githubusercontent.com/5713833/118035745-a05eb080-b339-11eb-8c72-9283525be61c.png)

Reply:
OK. Never mind. Makes more sense when you see all the rules.